src/main/kotlin/racing/CarRacingResultView.kt

```diff
@@ -1,21 +1,26 @@
 package racing
 
-class CarRacingResultView {
+object CarRacingResultView {
     fun printCarRacingResult(cars: List<Car>, moveCount: Int) {
         // turnOfPosition의 0번 index는 기본으로 1이 셋팅되어있기때문에
         // 범위를 0 <= moveIndex <= moveCount 로 설정
         for (moveIndex in 0..moveCount) {
-            cars.forEach { car ->
-                printCurrentCarMovingDistance(car.turnOfPosition[moveIndex])
-            }
-            println()
+            printMovingDistanceByTurn(cars = cars, moveIndex = moveIndex)
         }
     }
 
-    private fun printCurrentCarMovingDistance(carPosition: Int) {
-        for (i in 0 until carPosition) {
-            print("_ ")
+    private fun printMovingDistanceByTurn(cars: List<Car>, moveIndex: Int) {
+        cars.forEach { car ->
+            printCurrentCarMovingDistance(carName = car.name, carPosition = car.turnOfPosition[moveIndex])
         }
         println()
     }
+
+    private fun printCurrentCarMovingDistance(carName: String, carPosition: Int) {
+        println("$carName : ".plus("-".repeat(carPosition)))
```

Comment:
```suggestion
        println("$carName : ${"-".repeat(carPosition)})
```
가독성을 고려하면 문자열 템플릿이 더 나은것 같아요.